spec/models/donation_spec.rb

```diff
@@ -21,6 +22,24 @@
     end
   end
 
+  describe ".confirmed" do
+    it "includes confirmed donations" do
+      confirmed_donation = create(:donation, :confirmed)
+      reconfirmed_donation = create(:donation, declined_at: 1.week.ago, confirmed_at: 1.day.ago)
+      create(:donation, confirmed_at: 1.week.ago, declined_at: 1.day.ago)
+      create(:donation, :declined)
+      create(:donation, :pending)
+
+      confirmed = Donation.confirmed
+      donors = confirmed.map(&:donor)
+
+      expect(donors.map(&:name)).to eq([
+        confirmed_donation.donor.name,
```

Comment:
This current Hound default drives me batty. You can fix it by putting the array bracket one line down.